### ISSUE TITLE
Swap memory in bytes on Linux but in KB on MacOS 

### ISSUE THREAD

**Issue by FZambia** [shirou/gopsutil#299]
Hello!

I just noticed that on MacOS (10.11.6) `SwapMemoryStat` contains [kilobytes](https://github.com/shirou/gopsutil/blob/master/mem/mem_darwin.go#L60) for Total, Used, Free field values while on linux (Centos 6 in my case) those fields contain bytes.

Output of `sysctl -n vm.swapusage` on MacOS:
```
total = 2048.00M  used = 1458.25M  free = 589.75M  (encrypted)
```

Then on linux the following script (simplified version of how `gopsutil` checks swap mem on Linux):
```go
package main

import "syscall"
import "fmt"

func main() {
        sysinfo := &syscall.Sysinfo_t{}
        if err := syscall.Sysinfo(sysinfo); err != nil {
                panic( err)
        }
        fmt.Println(sysinfo.Unit)
        fmt.Println(sysinfo.Totalswap)
}
```

Outputs:
```
1
1073737728
```

**Comment by shirou** [comment]
Exactly! 

I fixed at 1123132e5ad392485dbd16e0f09a0db8c963a2ab to convert bytes, same as linux.
Thank you for your information!

**Comment by FZambia** [comment]
@shirou thanks for a very useful library!
